compiler/optimizer/DebuggingCounters.cpp

```diff
@@ -172,14 +172,14 @@ void TR_DebuggingCounters::insertCounter(const char * name, TR::Compilation * co
       // Treetops for counterRef = counterRef + 1;
 
       TR::Node * node = tt->getNode();
-      TR::Node* loadNode = TR::Node::createWithSymRef(node, TR::iuload, 0, counterRef);
+      TR::Node* loadNode = TR::Node::createWithSymRef(node, TR::iload, 0, counterRef);
 
       TR::Node* addNode =
          TR::Node::create(TR::iuadd, 2, loadNode,
 		         TR::Node::create(node, TR::iuconst, 0, 1));
```

Comment:
these should just be cleaned up at the same time - make the whole tree consistent iload to iadd with iconst 1 to istore.

Reply:
The `iuconst` and `iuadd` are separately being mapped to their normal versions in #4347 and #4366. Do you think I should change them at the same time or keep the current change(keep them separate).

Reply:
Ok we can keep the cleanliness here by separating the opcodes I guess.